bundles/org.openhab.binding.jablotron/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="jablotron"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- Oasis -->
+	<!-- StatusA -->
+	<channel-type id="statusA">
+		<item-type>Switch</item-type>
+		<label>Alarm Zone A Status</label>
+		<description>Status of the A zone of your alarm</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<!-- StatusB -->
+	<channel-type id="statusB">
+		<item-type>Switch</item-type>
+		<label>Alarm B Status</label>
+		<description>Status of the B zone of your alarm</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<!-- StatusABC -->
+	<channel-type id="statusABC">
+		<item-type>Switch</item-type>
+		<label>Alarm Zone ABC Status</label>
+		<description>Status of the ABC zone of your alarm</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<!-- StatusPGX -->
+	<channel-type id="statusPGX">
+		<item-type>Switch</item-type>
+		<label>OASIS PGX Status</label>
+		<description>Status of the PGX switch of your OASIS alarm</description>
+	</channel-type>
+
+	<!-- StatusPGY -->
+	<channel-type id="statusPGY">
+		<item-type>Switch</item-type>
+		<label>OASIS PGY Status</label>
+		<description>Status of the PGY switch of your OASIS alarm</description>
+	</channel-type>
+
+	<!-- JA100 -->
+	<channel-type id="status">
+		<item-type>Number</item-type>
+		<label>Alarm Section Status</label>
+		<description>Status of the section of your alarm</description>
+	</channel-type>
+
+	<!-- StatusPGM -->
+	<channel-type id="statusPGM">
+		<item-type>Switch</item-type>
+		<label>JA-100 PGM Status</label>
+		<description>Status of the PGM switch of your JA-100 alarm</description>
+	</channel-type>
+
+	<!-- Alarm -->
+	<channel-type id="alarm">
+		<kind>trigger</kind>
+		<label>Alarm</label>
+		<description>Status of the alarm of your alarm</description>
+		<state readOnly="true"/>
+	</channel-type>
```

Comment:
As this trigger Channel has payload, the `<event />` tag shall be specified. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#trigger-channel-types

Reply:
specified the events